lib/datasets_test.go

```diff
@@ -439,21 +517,72 @@ func TestDatasetRequestsGetFSIPath(t *testing.T) {
 		t.Fatalf("error checking out dataset: %s", err)
 	}
 
-	getParams := &GetParams{
-		Ref: "peer/movies",
-	}
-	got, err := inst.Dataset().Get(ctx, getParams)
+	var nilCommit *dataset.Commit
+
+	moviesDsAtPath := &dataset.Dataset{}
+	copier.Copy(moviesDsAtPath, moviesDs)
```

Comment:
is there a better way to deep copy a dataset?

Reply:
So looking into it, looks like `reflect.Copy` is only for slices/arrays/or copying a string into a []UInt8

Leaving this as us & adding an issue in `dataset` pitching a `dataset.Copy`.